libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -299,14 +301,30 @@ ASTPointer<ContractDefinition> ASTJsonImporter::createContractDefinition(Json::V
 	);
 }
 
+ASTPointer<IdentifierPath> ASTJsonImporter::createIdentifierPath(Json::Value const& _node)
+{
+	astAssert(_node["name"].isString(), "Expected 'name' to be a string!");
+
+	vector<ASTString> namePath;
+	vector<string> strs;
+	string nameString = member(_node, "name").asString();
+	boost::algorithm::split(strs, nameString, boost::is_any_of("."));
+	for (string s: strs)
+		namePath.emplace_back(s);
```

Comment:
```suggestion
		namePath.emplace_back(std::move(s));
```

Reply:
Actually I think it is fine to split directly into `namePath`.

Reply:
Unfortunately I am getting a very long list of errors when I try to pass it directly